src/notification.c

```diff
@@ -248,12 +280,32 @@ handle_remove_notification (XdpImplNotification *object,
   return TRUE;
 }
 
+static void
+check_notification_impl_version (void)
+{
+  guint activate_signal;
+  GSignalQuery activate_signal_query = {0};
+
+  activate_signal = g_signal_lookup ("action-invoked",
```

Comment:
Yuck.

Can't we use a proper version?

Reply:
IIRC there was no way to get at this level? Or I wanted to support both :)

I need to check the details...So long time :)